### ISSUE TITLE
Test Copper/coprace/coprace2: Strange OCS / ECS difference

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#643]
To get a grip on #631, I've broken down the test to a couple of simpler ones. To my surprise, I've arrived at a test that looks different on my OCS A500 and my ECS A500. The test itself is pretty simple. The CPU and the Copper run side by side. Both modify the background color. 

A500 OCS: 🥰
![coprace2_A500_OCS](https://user-images.githubusercontent.com/12561945/151712155-82261f4c-f90f-46b7-a92b-5c80666914ab.JPG)

A500 ECS: 😍
![coprace2_A500_ECS](https://user-images.githubusercontent.com/12561945/151712161-bad9ad55-7441-4f13-9770-44f6ec93148c.JPG)

The test is important because many of my other tests rely on precise CPU timing. Right now, I have no idea what could account for the difference.

The current version of vAmiga matches what I see on the ECS machine (assuming I haven't overseen anything), but I need to know why the OCS machine is different. 

<img width="976" alt="Bildschirmfoto 2022-01-30 um 19 25 22" src="https://user-images.githubusercontent.com/12561945/151712357-60049d7f-2cd9-46a8-8590-91b9de6c1c28.png">

**Comment by dirkwhoffmann** [comment]
Related to #711 and #709. 

I've modified the test such that one of the color register writes is routed to COLOR31: [coprace2b.zip](https://github.com/dirkwhoffmann/vAmiga/files/8933195/coprace2b.zip). This write is used as the trigger condition for the logic analyser: 

This may be the position where the OCS machines and the ECS machine behave differently (OCS is on top, ECS is below): 

![IMG_2986](https://user-images.githubusercontent.com/12561945/174429030-b44d86b1-81ba-47bb-b6da-7ff938ca950e.JPG)

I didn't analyse the traces in all detail, but I got the following impression:

- On the ECS machine, the negative edge on IPL0 always seems to occur slightly after the rising edge of CLK.
- On the OCS Rev. 3 machine, the negative edge on IPL0 sometimes occurs after, at the same time, or before the rising edge of CLK. 

I'm unsure if it's worth to debug this further. Maybe it's best to mark my Rev. 3 machine as "suspicious", put it into the shelf and replace it with my new Rev. 5 OCS machine.
